### PR TITLE
Enabling testsuite/integration/microprofile-tck/jwt to run against bo…

### PR DESCRIPTION
**Pull request by fabiobrz**
```
$ mvn clean test -Dts.bootable -Dmaven.repo.local=/home/fburzigo/projects/git/wildfly/wildfly/local-repo -DtestLogToFile=false -pl jwt/ -am
...

[INFO] Tests run: 113, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 24.731 s - in TestSuite
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 113, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for WildFly Test Suite: Integration - MicroProfile TCK 21.0.0.Beta1-SNAPSHOT:
[INFO] 
[INFO] WildFly Test Suite: Integration - MicroProfile TCK . SUCCESS [  2.602 s]
[INFO] WildFly Test Suite: Integration - MicroProfile - JWT SUCCESS [ 33.385 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  36.758 s
[INFO] Finished at: 2020-08-17T16:09:17+02:00
[INFO] ------------------------------------------------------------------------

```

### CODE REVIEW

**Comment by fabiobrz** [comment]
Hi @honza-kasik - I've done this quickly and was wondering whether you'd have a look as well. WDYT?

File: testsuite/integration/microprofile-tck/jwt/pom.xml [hunk 1]
```diff
@@ -182,6 +182,113 @@
...
+                                    <goal>provisioning</goal>
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-jar-maven-plugin</artifactId>
+                        <version>${version.org.wildfly.jar.plugin}</version>
```
Comment (honza-kasik) [inline testsuite/integration/microprofile-tck/jwt/pom.xml:233]: I don't see this version defined anywhere. Or am I missing something?

File: testsuite/integration/microprofile-tck/jwt/pom.xml [hunk 2]
```diff
@@ -182,6 +182,113 @@
...
+                <property>
+                    <name>ts.bootable</name>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.wildfly.arquillian</groupId>
+                    <artifactId>wildfly-arquillian-container-bootable</artifactId>
+                    <scope>test</scope>
+                </dependency>
```
Comment (honza-kasik) [inline testsuite/integration/microprofile-tck/jwt/pom.xml:198]: Is this dependency defined in parent pom?

**Review by honza-kasik — COMMENTED** [review]
Looks good to me, I just have two questions.

File: testsuite/integration/microprofile-tck/jwt/pom.xml (hunk 1 above)
Comment (fabiobrz) [reply testsuite/integration/microprofile-tck/jwt/pom.xml:233]: It was added on the topic branch that this PR is targeting, see https://github.com/fabiobrz/wildfly/blob/adjust-ts-for-bootable-jar/pom.xml#L235

File: testsuite/integration/microprofile-tck/jwt/pom.xml (hunk 2 above)
Comment (fabiobrz) [reply testsuite/integration/microprofile-tck/jwt/pom.xml:198]: Yep, here it is: https://github.com/fabiobrz/wildfly/blob/adjust-ts-for-bootable-jar/pom.xml#L8644

File: testsuite/integration/microprofile-tck/jwt/pom.xml (hunk 1 above)
Comment (honza-kasik) [reply testsuite/integration/microprofile-tck/jwt/pom.xml:233]: @fabiobrz Sorry, I missed this is not against master!

**Comment by honza-kasik** [comment]
LGTM! Thank you! Merging...

**Pull request merged**